SRC/MutateFactory.cpp

```diff
@@ -71,7 +37,7 @@ void MutateFactory::initMutate()
 			{
 				mp_FileOp->init();
 				mp_factory->SetOperationType(configStruct.op_type);
-				mp_MutateOp = mp_factory->getOperation();
+				mp_MutateOp = mp_factory->getOperationType();
 				PerformMutationTesting(configStruct.op_type);

```

Comment:
Can we handle exceptions like Ctrl+Z or Ctrl+C ?

Reply:
Handling exception by signal handling is done in new PR